VideoWeb/VideoWeb.UnitTests/Helpers/ParticipantTilePositionHelperTest.cs

```diff
@@ -42,8 +28,22 @@ private List<ParticipantResponse> GetParticipantResponses()
                 new ParticipantResponse { Id = Guid.NewGuid(), Role = Role.JudicialOfficeHolder, DisplayName = "Part6", CaseTypeGroup = "group1", HearingRole = "Witness" },
                 new ParticipantResponse { Id = Guid.NewGuid(), Role = Role.Individual, DisplayName = "Part7", CaseTypeGroup = "group1", HearingRole = "Applicant" }
             };
+        }
+
+        [Test]
+        public void Should_set_tailed_display_names_for_non_judge_participants_number_participant()
+        {
+            ParticipantTilePositionHelper.AssignTilePositions(_participants);
+
+            _participants[0].TiledDisplayName.Should().Be($"JUDGE;HEARTBEAT;{_participants[0].DisplayName};{_participants[0].Id}");
```

Comment:
We should either assert if the display name property contains the participant display name and id, or we can store this magic string in a separate const and leave the assertions as is

Reply:
The string has to be in the correct order so doing a contain isn't specific enough. What do you mean by "can store this magic string in a separate const and leave the assertions as is"?

Reply:
It looks like the first part of this string is related to the role that we get from pariticpantResponse, can we reference the role here instead? 
For the heartbeat string we can probably declare two seperate consts and reference it here, so basically it will be like this:
$"{_participants[0].Role.toUppercase()};{heartbeat};{_participants[0].DisplayName};{_participants[0].Id}"

But do we need to unit test the complete TiledDisplayName here? Can we just check for the variables being displayed here instead, so using the Contain to check for the properties? 

_participants[0].TiledDisplayName.Should().Contain(_participants[0].DisplayName);

Reply:
The role doesn't directly relate to that Role enum i.e. Individual -> Civilian; JOH -> Civilian. I have raised a refactor ticket about the ParticipantTilePositionHelper as well.

Yeah as this class has to be correct as it is what populates it for the front end. If the ID and Display name were the wrong way round. None of Kinly's events would work and the display name would be the participants ID. Using contains doesn't check the order of the role, heartbeat mode, display name and ID.